src/content/docs/agents/java-agent/getting-started/introduction-new-relic-java.mdx

```diff
@@ -49,6 +49,38 @@ To [view your app's performance in the New Relic UI](/docs/apm/applications-menu
 
 You can also [extend your agent instrumentation](#extend-instrumentation) and explore your data with other New Relic products, including New Relic One's [data explorer](/docs/query-your-data/explore-query-data/data-explorer/introduction-data-explorer) and [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-new-relic-one-dashboards).
 
+## Evaluate our Java agent with guided install [#guided-install]
+
+Our guided install creates a customized CLI command for your environment that downloads and installs the Java agent. 
+
+<Callout variant="caution">
+The guided install version of the Java agent is for evaluation purposes only. It will not persist when you restart your Java application.
+</Callout> 
+
+![A screenshot of the guided install CLI.](./images/guided-install-cli.png "The guided install CLI.")
+
+Ready to get started? Click one of these buttons to try it out.
```

Comment:
"click one" makes it seem like anyone could do either, which isn't really the case. Maybe change this to, "Ready to get started? Click the Guided install button. If your account reports data through our [EU datacenter](link to datacenter doc), Click on the EU Guided install."

Reply:
Great suggestion!